src/sass/partials/layouts/_content-card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 2rem 0 0;
+  width: 100%;
+
+
+  @include breakpoint(setting-get('two grid cards')) {
+    margin-right: 4%;
```

Comment:
No sudden change, I still new to the base level styling that you guys have done in the past. No worries, I will adjust to make it consistent.